src/clp_ffi_py/ir/native/PyDeserializerBuffer.cpp

```diff
@@ -376,6 +376,21 @@ auto PyDeserializerBuffer::test_streaming(uint32_t seed) -> PyObject* {
 PyObjectStaticPtr<PyTypeObject> PyDeserializerBuffer::m_py_type{nullptr};
 PyObjectStaticPtr<PyObject> PyDeserializerBuffer::m_py_incomplete_stream_error{nullptr};
 
+auto PyDeserializerBuffer::create(PyObject* input_stream, Py_ssize_t buf_capacity)
+        -> PyDeserializerBuffer* {
+    // NOLINTNEXTLINE(cppcoreguidelines-pro-type-cstyle-cast)
```

Comment:
do we still need this since next line doesn't have any cast?

Reply:
`PyObject_New` is a macro and it uses C-style casting (welcome to CPython, lol)

Reply:
GG, learned